cmd/cat-main.go

```diff
@@ -105,6 +162,13 @@ func catOut(r io.Reader, size int64) *probe.Error {
 	var n int64
 	var e error
 
+	// In case of a user showing the object content in a terminal,
+	// avoid printing control and other bad characters to avoid
+	// terminal session corruption
+	if terminal.IsTerminal(int(os.Stdout.Fd())) {
```

Comment:
@vadmeste I used `cat -A` several times to view text files to know DOS format and tab character.  I have no experience in binary files with that.  We could intelligently print binary files with hex or other encoding.  If we don't go with `cat -A` behavior, someone can use it like `mc cat ... | cat -A`

I see `less` and `cat -A` show same output.